src/Tizen.NUI.Components/Controls/Picker.cs

```diff
@@ -413,10 +415,43 @@ private void Initialize()
 
             Focusable = true;
             KeyEvent += OnKeyEvent;
+
+            Accessibility.Accessibility.ScreenReaderEnabled += (s, e) => {
```

Comment:
You do not have to call ```GrabAccessibilityHighlight``` and ```ClearAccessibilityHighlight```,
and you might need to call ```UpdateAccessbilityHiddenItemList``` when the ```ScreenReaderEnabled```.

Reply:
that's right. i missed it.